### PR TITLE
Update docs about not supported rhel 7.5 and 7.6

### PR DESCRIPTION
**Pull request by gnufied**
We don't support those versions because QEMU is too old.

### CODE REVIEW

File: docs/dev/libvirt-howto.md [hunk 1]
```diff
@@ -19,12 +19,14 @@ crw-rw-rw-+ 1 root kvm 10, 232 Oct 31 09:22 /dev/kvm
 If it is missing, try some of the ideas [here][kvm-install].
 
 ### Install and Enable Libvirt
-On Fedora, CentOS/RHEL:
+On Fedora or RHEL systems with QEMU(2.4+):
```
Comment (wking) [inline docs/dev/libvirt-howto.md]: I think floor versions like this belong in `docs/dev/dependencies.md` under a new subsection:

```markdown
## Runtime Dependencies

...FIXME Terraform...

### Libvirt

Versions of QEMU less than 2.4 lack support for `-fw_cfg`, which is required for Ignition.
QEMU 2.4 and later may or may not work, we're currently getting testing in place to check at least some versions.
```

Or something very roughly like that.  Thoughts?

Comment (gnufied) [reply docs/dev/libvirt-howto.md]: Sure we can document that there too. But I did wanted to mention that, if somehow your RHEL install does have qemu version 2.4+, the installer may work.

**Comment by leifmadsen** [comment]
This just bit me, and it wasn't clear at all that the issue was an older version of QEMU. Instead of spending 20+ mins waiting for images to upload and complete when the QEMU (and other software versions) is too old, it would be nice if the installer would just do some checks to validate that requirements for a successful install have been completed.

**Comment by e-minguez** [comment]
Just in case, I've opened a bugzilla to track it here https://bugzilla.redhat.com/show_bug.cgi?id=1674422

**Comment by abhinavdahiya** [comment]
closing due to inactivity. Please reopen if needed.

/close

**Comment by sbuvaneshkumar** [comment]
> Just in case, I've opened a bugzilla to track it here https://bugzilla.redhat.com/show_bug.cgi?id=1674422

@e-minguez I see the BZ closed as not a bug.

I have burn almost an hr to setup this in CentOS 7.8.
I think it is important to update the doc.

**Pull request closed without merging**